src/components/Buttons/LinkButton.tsx

```diff
@@ -1,31 +1,21 @@
-"use client";
-
 import React from "react";
-import Link from "next/link";
-import styled from "styled-components";
 import Button from "@mui/material/Button";
-import { usePathname } from "next/navigation";
-
-const UnstyledLink = styled(Link)`
-    text-decoration: none;
-    display: contents;
-`;
+import { ButtonWrap, UnstyledLink } from "@/components/Buttons/GeneralButtonParts";
 
-type LinkButtonProps = {
+interface Props {
+    children: string;
     link: string;
-    page: string;
-    onClick?: () => void;
-};
+}
 
-const LinkButton: React.FC<LinkButtonProps> = (props) => {
-    const pathname = usePathname();
-    const active = pathname?.startsWith(props.link) ?? false;
+const LinkButton: React.FC<Props> = ({ children, link }) => {
     return (
-        <UnstyledLink key={props.page} href={props.link} prefetch={false}>
-            <Button color="primary" variant={active ? "contained" : "outlined"}>
-                {props.page}
-            </Button>
-        </UnstyledLink>
+        <ButtonWrap>
+            <UnstyledLink href={link} prefetch={false}>
```

Comment:
Do we always want `prefetch={false}`

Reply:
discussed in person - will change all the prefetch to be true (by default)